schemas/common/extensible.schema.json

```diff
@@ -50,7 +50,10 @@
             "^dsp:.*$": {},
             "^searchads:.*$": {},
             "^adcloud:.*$": {},
-            "^id3:.*$": {},
+            "^sfdc:.*$": {},
```

Comment:
Where did these other namespaces show up?  I haven't seen them used anywhere...

Reply:
As previously told, we had intentions to add these initially but later we decided not. Deleted these now,